src/xtgeo/common/log.py

```diff
@@ -30,3 +32,30 @@ def null_logger(name: str) -> logging.Logger:
     logger = logging.getLogger(name)
     logger.addHandler(logging.NullHandler())
     return logger
+
+
+def functimer(func):
```

Comment:
Not sure if correct place, or if it needed, but is was quite useful in the development phase.